dunning_kruger.Rmd

```diff
@@ -103,3 +103,7 @@ df
 ```{python}
 df.groupby('actual_quantile', observed=True)['difference'].mean()
 ```
+
+```{python}
+# an arbitrary change
```

Comment:
This change seems a bit - well - arbitrary - can you make a more substantial change?

Reply:
OK I will have a think about other changes.